pkg/cmd/server/origin/master.go

```diff
@@ -148,24 +148,42 @@ func (c *MasterConfig) BuildClients() {
 	c.osClient = osclient
 }
 
+// KubeClient returns the kubernetes client object
 func (c *MasterConfig) KubeClient() *kclient.Client {
 	return c.kubeClient
 }
+
+// OSClient returns the openshift client object
+func (c *MasterConfig) OSClient() *osclient.Client {
```

Comment:
starts using what others?  i'll change it, but i think i need some education on the vision here.